packages/bpk-component-badge/BadgeLayout.js

```diff
@@ -35,7 +35,7 @@ const LIGHT_BADGES = [
 
 export type Props = {
   docked: ?string,
-  children: Element<*>,
+  children: Node,
```

Comment:
Flow was complaining when I put an array of `div`s inside a `BadgeLayout`, this fixes that.

Reply:
Yeah I think this is the correct way of doing it!